docs/Makefile

```diff
@@ -1,14 +1,23 @@
-.PHONY: docs docs-test
+.PHONY: docs docs-test docs-no-pull docs-generate-frontend docs-no-generate-frontend
 
 IMAGE = grafana/docs-base:latest
 
-docs:
-	docker pull ${IMAGE}
+docs-generate-frontend:
+	yarn install --pure-lockfile
+	yarn run packages:build
+	yarn run packages:docsExtract
+	yarn run packages:docsToMarkdown
+	yarn run packages:clean
```

Comment:
Already approved, but got a question, can we move this to a `yarn` script to make things easier? Then CI could also use the same `yarn` script.

Reply:
hmm, yeah - I think that should be possible. Maybe do it in a follow up PR? If that is ok?

Reply:
Yeah sure, fine by me.